crowdsourcing/viewsets/project.py

```diff
@@ -89,11 +89,25 @@ def list(self, request, *args, **kwargs):
         SELECT m.*, weight, average_worker_rating, imputed_rating FROM crowdsourcing_module m
             INNER JOIN crowdsourcing_requester r ON m.owner_id = r.id
             INNER JOIN crowdsourcing_userprofile u ON r.profile_id = u.id
-            LEFT OUTER JOIN crowdsourcing_workerrequesterrating w
-                ON u.id = w.origin_id AND w.type='requester' AND w.target_id=%s
-            LEFT OUTER JOIN (SELECT target_id,  AVG(weight) AS average_worker_rating  FROM
-                crowdsourcing_workerrequesterrating
-                WHERE type='requester' AND target_id=%s GROUP BY target_id) tmp ON TRUE
+            LEFT OUTER JOIN 
+                (SELECT w.* FROM crowdsourcing_workerrequesterrating w
+                INNER JOIN(
+                    SELECT origin_id, MAX(last_updated) AS max_date FROM crowdsourcing_workerrequesterrating
+                        WHERE type='requester' AND target_id = %s GROUP BY origin_id) tb
+                    ON w.origin_id = tb.origin_id AND w.last_updated = tb.max_date
+                    AND w.type='requester' AND w.target_id=%s) w
+                ON u.id = w.origin_id
+            LEFT OUTER JOIN (
+                SELECT target_id, AVG(weight) AS average_worker_rating from
+                (SELECT wr.* FROM crowdsourcing_workerrequesterrating wr
+                INNER JOIN (
+                    SELECT origin_id, target_id, MAX(last_updated) AS max_date
```

Comment:
so i think that makes a little bit less sense in the cascade than in the task feed. but yes i suppose we could exclude this user acting as a requester rating himself as a worker. 

thinking it through more, it is impossible because we dont allow users (as workers) to accept tasks they posted as requesters and therefore garner ratings from themselves. so shouldnt be a problem? what do you think?